blobstore/src/main/java/org/jclouds/blobstore/config/LocalBlobStore.java

```diff
@@ -878,7 +888,7 @@ public String completeMultipartUpload(MultipartUpload mpu, List<MultipartPart> p
 
       setBlobAccess(mpu.containerName(), mpu.blobName(), mpu.putOptions().getBlobAccess());
 
-      return eTag;
+      return mpuETag;
```

Comment:
Does this return an MPU ETag for single-part uploads?

Reply:
This will return an MPU for any multi-part uploads, even if they are composed of one part. This matches the AWS S3 behavior.

Reply:
Sorry I missed that this was party of `completeMultipartUpload`.